Tools/Scripts/filter-test-logs

```diff
@@ -41,7 +42,6 @@ def parser():
 
     return parser.parse_args()
 
-
```

Comment:
Normal Python style is two lines between top-level blocks, so plz don't delete this.